pennylane/_device.py

```diff
@@ -62,12 +62,20 @@ def __init__(self, wires=1, shots=1000):
 
     def __repr__(self):
         """String representation."""
-        return "{}.\nInstance: ".format(self.__module__, self.__class__.__name__, self.name)
+        return "<{} ({}, wires={}, shots={}) at {}>".format(
+            self.__class__.__name__, self.short_name, self.num_wires, self.shots, hex(id(self))
```

Comment:
I always though "at 0xABD..." referred to the memory address. Turns out it doesn't!

Reply:
Same here. I just used `id` originally, and was wondering why there was no `x`!